### PR TITLE
Add reloads collector

### PR DESCRIPTION
**Pull request by gondaruk**
This PR adds a `reloadsCollector` which is supposed to collect `reloads` from logstash node api, which contains a number of successful and failed configuration reload attempts.

### CODE REVIEW

File: collector/reloads_config.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+func newReloadsConfigCollector() *reloadsConfigCollector {
+	desc := newDescFunc(namespace, "reloads_config")
+	return &reloadsConfigCollector{
+		Failures:  desc("failures", "Number of failures during config reload."),
+		Successes: desc("successes", "Number of successful config reloads."),
+	}
+}
+
+func (c *reloadsConfigCollector) Collect(p ReloadsConfig, ch chan<- prometheus.Metric) {
+	ch <- prometheus.MustNewConstMetric(c.Failures, prometheus.GaugeValue, float64(p.Failures))
```
Comment (Wing924) [inline collector/reloads_config.go]: I think the type of `successes` and `failures` are counter.
How about changing it to prometheus.CounterValue?

File: collector/reloads_config.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+type reloadsConfigCollector struct {
+	Failures  *prometheus.Desc
+	Successes *prometheus.Desc
+}
+
+func newReloadsConfigCollector() *reloadsConfigCollector {
+	desc := newDescFunc(namespace, "reloads_config")
+	return &reloadsConfigCollector{
+		Failures:  desc("failures", "Number of failures during config reload."),
```
Comment (Wing924) [inline collector/reloads_config.go]: I think the type of successes and failures are counter.
How about changing it to failures_total and successes_total?

**Review by Wing924 — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Just a small issue here.

File: collector/reloads_config.go (hunk 2 above)
Comment (gondaruk) [reply collector/reloads_config.go]: Definitely makes sense to me, thanks.

File: collector/reloads_config.go (hunk 1 above)
Comment (gondaruk) [reply collector/reloads_config.go]: Nice catch, thank you!

**Comment by gondaruk** [comment]
Thanks for reviewing. I've refreshed the PR.
Please let me know if it looks better to you now. Thanks in advance!

**Review by Wing924 — APPROVED** [review]

**Pull request merged**